shimmingtoolbox/masking/shape.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+"""
+Image mask with shape API
+"""
+
+import numpy as np
+
+
+def shape(data, shape, center_x=None, center_y=None, center_z=None, len_x=None, len_y=None, len_z=None, radius=None):
```

Comment:
I think a wrapper function would be a good way to go. Normally when using wrapper function like the one you have, you should use *args and **kargs and then for each call, check if the correct arguments are passed down to the wrapper. This way, it can be used with any type of function. And could even be implemented as a decorator, but it is not necessary:
```suggestion
def shape(data, shape, **kargs):
    mask_info = {}
    for key, value in kargs.items():
        mask_info[key] = value
```
Then you would have a dictionary with all the information provided and you can parse through it to check if the required fields for your shape are present.